src/main/java/fi/metatavu/kuntaapi/server/index/IndexableCode.java

```diff
@@ -0,0 +1,95 @@
+package fi.metatavu.kuntaapi.server.index;
+
+public class IndexableCode implements Indexable {
+
+  @Field (index = "not_analyzed", store = true, type = "long")
+  private Long orderIndex;
+  
+  @Field (index = "not_analyzed", store = true)
+  private String codeId;
+  
+  @Field (index = "not_analyzed", store = true)
+  private String codeType;
+  
+  @Field (index = "not_analyzed", store = true)
+  private String code;
+
+  @Field (analyzer = "finnish")
+  private String nameFi;
+
+  @Field (analyzer = "swedish")
+  private String nameSv;
+
+  @Field (analyzer = "english")
+  private String nameEn;
+  
+  protected IndexableCode() {
+  }
+  
+  public IndexableCode(Long orderIndex, String codeId, String codeType, String code, String nameFi, String nameSv, String nameEn) {
+    super();
+    this.orderIndex = orderIndex;
+    this.codeId = codeId;
+    this.codeType = codeType;
+    this.code = code;
+    this.nameEn = nameEn;
+    this.nameSv = nameSv;
+    this.nameFi = nameFi;
+  }
+
+  @Override
+  public String getId() {
+    return codeId;
+  }
+
+  @Override
+  public String getType() {
+    return "code";
+  }
+
+  @Override
+  public Long getOrderIndex() {
+    return orderIndex;
+  }
+  
+  public void setOrderIndex(Long orderIndex) {
+    this.orderIndex = orderIndex;
+  }
+  
+  public String getCode() {
+    return code;
+  }
+  
+  public String getCodeId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 41. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)